package FinalExam.oop;

import java.io.IOException;

public class famApp {
    public static void main(String[] args) {
   fam f=new fam1("q");
   fam1 d=new fam1("q");
  // f=d;
        System.out.println(f.equals(d));

    }
    class MultipleCatchBlock5{
        public static void main(String args[])throws Exception
        {
  
            System.out.println("rest of the code...");
        }
    }
}
